packages/next/build/index.ts

```diff
@@ -193,6 +194,19 @@ export default async function build(
       typeCheckingSpinner.stopAndPersist()
     }
 
+    if (config.experimental.eslint) {
+      await nextBuildSpan
+        .traceChild('verify-and-lint')
+        .traceAsyncFn(async () => {
+          verifyAndLint(
```

Comment:
This is not `await`ed, which means the trace will instantly resolve (0ms) instead of showing the timing.

Reply:
Fixed 👍